.cspell.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
🎨 Check out https://github.com/bitwarden/template/pull/14/files#diff-4e0f175c840784de9e935f881821905c75a2a125aa57c51268a7fb4496edfbe6 which I got from their site with schema, ignore paths, a separate file for words, etc. -- what do we want to recommend here?

Reply:
I'm fine with most of that, especially the json schema, but I don't like the project dictionary for two reasons:
1. more root level files is annoying and I'm not clear what the gain is
2. the `flagWords` concept is useful (see [here](https://github.com/bitwarden/autopush-manager/blob/70837aec0ac0b784dd00ff85d645e742910e25ef/.cspell.json#L8)) and there isn't a similar concept where you have a flagDictionary.  We need people to get in this file anyways.